### ISSUE TITLE
[USE CASE]: Document android usage

### ISSUE THREAD

**Issue by michaelneale** [TBD54566975/developer.tbd.website#1262]
### Developer Value Added

Android is a super popular platform and tbdex can work on it quite well

### Use Case

Document an example and any "gotchas" for using tbdex-kt in android, and link to an example repo: https://github.com/TBD54566975/tbdex-android-example

### Dependencies

Ideally the AndroidKeyManager implementation woudl be in web5-kt to make it simple: https://github.com/TBD54566975/web5-kt/issues/250 but this can be optional

**Comment by chris-tbd** [comment]
@michaelneale when we're talking about potential "gotchas" for the Android Platform, what did you run into? I see that the `InMemoryKeyManager` may not work, but outside of that was there anything in the SDK or setup that wasn't working out of the gate? Or, was the main value to share about how to get the SDK integrated into your project (as well as your "Android specific build tips" section)?

**Comment by ALRubinger** [comment]
On it. A lot of the open questions here can be solved atop the build work I'm doing in the `*-kt` projects; will be moving @michaelneale `AndroidKeyManager` into an Android extension for `webt-kt`, and creating a separate distro for that which does the right exclusions that Android can't bring in.

The nice test for this will be if Mic's examplar app can bring in one dependency, require no other weird build magic, and get everything it needs to Just Work.

Then can work with DevRel to get this documented.

**Comment by chris-tbd** [comment]
Can we create a separate issue for documenting `AndroidKeyManager` and/or migrating it into `web5-kt`?

**Comment by ALRubinger** [comment]
Absolutely @chris-tbd - just hadn't gotten to it yet. Feel free to drop an issue here for documenting it. I'll handle in `web5-kt` here:

* https://github.com/TBD54566975/web5-kt/issues/287

**Comment by angiejones** [comment]
@ALRubinger Chris has documented the current usage [here](https://developer.tbd.website/docs/tbdex/wallet/required-sdks?lang=Kotlin#install-sdks).

**Comment by michaelneale** [comment]
@chris-tbd yeah answered above, but mostly just packaging. I did build a key manager which was pretty simple as mentioned, but other than that it nicely just worked.
